addons/account/tests/test_fiscal_position.py

```diff
@@ -308,6 +308,13 @@ def test_fiscal_position_constraint(self):
             'zip_to': '456',
         }])
 
+    def test_fiscal_position_different_vat_country(self):
```

Comment:
No assert ? What behavior/error/exception/etc are we expecting ? (at least comment)